src/debug.ts

```diff
@@ -128,6 +152,35 @@ function debug(this: any, options: any) {
     reply()
   })
 
+  this.add('role:seneca,plugin:debug,cmd:toggle', function(this: any, _msg: any, reply: any) {
```

Comment:
do not use a "toggle" - this is very brittle
instead, make the state explict:
`sys:debug,area:trace,active:true|false`

here, `area` refers to the feature area - tracing - in future there may be others